tests/examples/start-message/playground/start-message/start-msg

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+# Not actually, but to demonstrate
+echo "Service has started at localhost:9001" 1>& 2
```

Comment:
That's some funky syntax.
I'm actually not sure what that will do.

Usually it would  be written `echo >&2`.